libyul/optimiser/StructuralSimplifier.cpp

```diff
@@ -93,26 +93,31 @@ void StructuralSimplifier::operator()(Block& _block)
 
 void StructuralSimplifier::simplify(std::vector<yul::Statement>& _statements)
 {
-	util::GenericVisitor visitor{
-		util::VisitorFallback<OptionalStatements>{},
```

Comment:
Out of curiosity: does it still happen, if you replace ``util::VisitorFallback<OptionalStatements>{}`` with ``[](auto&) -> OptionalStatements { return {}; }``?

Reply:
Yes, it still crashes.